deptrack-devops/src/Devops/Graph.hs

```diff
@@ -148,6 +152,46 @@ type Broadcast = (OpUniqueId,CheckResult,Stability,WantedDirection) -> IO ()
 noBroadcast :: Broadcast
 noBroadcast = const (return ())
 
+-- | Turn-up a graph sequentially
+-- Topologically sort all nodes of the graph then run corresponding operations
+syncTurnupGraph :: Broadcast -> OpGraph -> IO ()
+syncTurnupGraph bcast (graph,lookupVertex,_) =
+  mapM_ go (reverse $ Graph.topSort graph)
```

Comment:
Wondering whether we should use `Graph.topSort (Graph.transposeG graph)` . In particular, if graph is using a Vector with o(n) transpose we'll avoid the o(n2) reverse. Also, maths-wise (this is subjective) I believe that it requires fewer "brain cells" to convince oneself that the topSort of a transposed graph is as valid as `reverse . topSort`.